portal/templates/redesign/teach_new/base_registering.html

```diff
@@ -14,4 +16,56 @@
 {% endblock rightMenuButtons %}
 
 {% block content %}
+<div class="banner banner--teacher--plain">
+    <h1 class="col-sm-offset-2">Welcome, {{ user|make_into_username }}</h1>
+    <h4 class="col-sm-offset-2">Everything you need to start coding with your class is here. Let&rsquo;s set you up.</h4>
+    <div class="arrows">
+        {% block setupCompleteStep %}
+        <div class="onboarding__step onboarding__step--todo">
+            <div class="onboarding__step__text">
+                <img title="To do" alt="Step to do" src='{% static "portal/img_new/step_undone.png" %}'>
+                <p class="body-text">Setup complete</p>
+            </div>
+        </div>
+        {% endblock setupCompleteStep %}
+        {% block printStudentCardsStep %}
+        <div class="onboarding__step onboarding__step--todo">
```

Comment:
Here is the power of that approach: if the images are not inline, all you need to control with your case logic is the  text itself and the modifier class.  And if you're externalising the text to internationalise, there may be a cleverer way of doing the text than a block.